crates/libduckdb-sys/build.rs

```diff
@@ -160,8 +160,7 @@ mod build_bundled {
             .flag_if_supported("-stdlib=libstdc++")
             .flag_if_supported("/bigobj")
             .warnings(false)
-            .flag_if_supported("-w")
-            .flag_if_supported("/D_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR");
+            .flag_if_supported("-w");
 
         if win_target() {
             cfg.define("DUCKDB_BUILD_LIBRARY", None);
```

Comment:
@Mytherin would the workaround suggested in the thread about runner images be still applicable? 
```suggestion
            cfg.define("DUCKDB_BUILD_LIBRARY", None);
            cfg.define("_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR", None);
```